docs/configuration-reference.md

```diff
@@ -94,13 +94,13 @@ Opsdroid comes with some built-in connectors out of the box. A connector is a mo
 
 The built-in connectors are:
 
-- [Facebook](/connectors/facebook.md)
-- [GitHub](/connectors/github.md)
-- [Matrix](/connectors/matrix.md)
-- [Rocket.Chat](/connectors/rocketchat.md)
-- [Slack](/connectors/slack.md)
-- [Telegram](/connectors/telegram.md)
-- [Websockets](/connectors/websocket.md)
+- [Facebook](connectors/facebook.md)
+- [GitHub](connectors/github.md)
+- [Matrix](connectors/matrix.md)
+- [Rocket.Chat](connectors/rocketchat.md)
+- [Slack](connectors/slack.md)
+- [Telegram](connectors/telegram.md)
```

Comment:
Not entirely sure if removing the `/` is the right thing to do here as the links will probably stop working